### PR TITLE
DAOS-10038 doc: Add concurrent pool creation warning

### PR DESCRIPTION
**Pull request by knard-intel**
# Description

Add documentation indicating that dmg is not supporting concurent creation of pools with size given as a percentage of the overall available storage size.

# Actions

## Before requesting gatekeeper:

* [ ] Two review approvals and any prior change requests have been resolved.
* [x] Testing is complete and all tests passed or there is a reason documented in the PR why it should be force landed and forced-landing tag is set.
* [x] Commit messages follows the guidelines outlined [here](https://daosio.atlassian.net/wiki/spaces/DC/pages/11133911069/Commit+Comments).
* [x] Any tests skipped by the ticket being addressed have been run and passed in the PR.

## Gatekeeper:

* [ ] You are the appropriate gatekeeper to be landing the patch.
* [ ] The PR has 2 reviews by people familiar with the code, including appropriate watchers.
* [ ] Githooks were used. If not, request that user install them and check copyright dates.
* [ ] Checkpatch issues are resolved.  Pay particular attention to ones that will show up on future PRs.
* [ ] All builds have passed.  Check non-required builds for any new compiler warnings.
* [ ] Sufficent testing is done. Check feature pragmas and test tags and that tests skipped for the ticket are run and now pass with the changes.
* [ ] If applicable, the PR has addressed any potential version compatibility issues.
* [ ] Check the target branch.   If it is master branch, should the PR go to a feature branch?  If it is a release branch, does it have merge approval in the JIRA ticket.
* [ ] Extra checks if forced landing is requested
  * [ ] Review comments are sufficiently resolved, particularly by prior reviewers that requested changes.
  * [ ] No new NLT or valgrind warnings.  Check the classic view.
  * [ ] Quick-build or Quick-functional is not used.
* [ ] Fix the commit message upon landing. Check the standard [here](https://daosio.atlassian.net/wiki/spaces/DC/pages/11133911069/Commit+Comments). Edit it to create a single commit. If necessary, ask submitter for a new summary.

### CODE REVIEW

File: docs/admin/pool_operations.md [hunk 1]
```diff
@@ -104,6 +104,13 @@ The capacity of the pool can be specified in three different ways:
     So in the first example above, specifying `--scm-size=256GB`
     would fail as 256 GB is smaller than the minimum 256 GiB.
 
+!!! warning
+    Concurrent creation of pools using **size percentage** could lead to
+    `ENOSPACE` errors.  Indeed, this operations is not atomic and the overall
```
Comment (kccain) [inline docs/admin/pool_operations.md:109]: Minor suggested change (up to you & other reviewers on this one):, I propose:

"Indeed, these operations are not atomic and the overall
available size retrieved in a first step could be different from the size
actually available when the second step will be performed (i.e. allocation of space
for the pool)."

Comment (kccain) [inline docs/admin/pool_operations.md:109]: I think these words should change to "these operations are"

**Review by kccain — CHANGES_REQUESTED** [review]

File: docs/admin/pool_operations.md (hunk 1 above)
Comment (mjmac) [reply docs/admin/pool_operations.md:109]: Agree with this change. Minor nit: "retrieved in a" -> "retrieved in the".

Comment (knard-intel) [reply docs/admin/pool_operations.md:109]: Thanks for your review and to correct my frenglish :flushed:

- [ ] Fix frenglish sentence

**Review by kccain — DISMISSED** [review]

**Review by mjmac — APPROVED** [review]
Looks good to me. I suggest changing the commit subject, though, as without context it seems to imply that the commit is actually adding concurrent creation support. Perhaps something like, "Add concurrent pool creation warning".

**Comment by knard-intel** [comment]
> Looks good to me. I suggest changing the commit subject, though, as without context it seems to imply that the commit is actually adding concurrent creation support. Perhaps something like, "Add concurrent pool creation warning".

- [ ]  Change commit title

**Comment by knard-intel** [comment]
Rebase was done for changing the commit title.
The content, the order, etc. of the commit were not updated and thus the commit history has been preserved.

**Comment by mjmac** [comment]
> Rebase was done for changing the commit title. The content, the order, etc. of the commits were not updated and thus the commit history has been preserved.

Just for future reference, a rebase is not necessary... You can just click the Edit button up at the top of the PR and change the subject. It will be used when the gatekeeper merges the PR.

**Review by kccain — APPROVED** [review]

**Comment by knard-intel** [comment]
> > Rebase was done for changing the commit title. The content, the order, etc. of the commits were not updated and thus the commit history has been preserved.
> 
> Just for future reference, a rebase is not necessary... You can just click the Edit button up at the top of the PR and change the subject. It will be used when the gatekeeper merges the PR. I just did it now.

OK, thanks for the tips :)

**Review by johannlombardi — APPROVED** [review]

**Pull request merged**